lib/graphoid/mutations/create_or_update.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module Graphoid
+  module Mutations
+    module CreateOrUpdate
+      def self.build(model)
+        Graphoid.initialize
+        grapho = Graphoid.build(model)
+        type = ::Types::MutationType
+
+        name = "create_or_update_#{grapho.name}"
+        plural_name = name.pluralize
+
+        type.field(name: name, type: grapho.type, null: true) do
+          argument :where, grapho.filter, required: true
+          argument :data, grapho.input, required: false
+        end
+
+        type.class_eval do
+          define_method :"#{name}" do |where: {}, data: {}|
+            begin
+              user = context[:current_user]
+
+              object = if model.respond_to?(:resolve_one)
+                          model.resolve_one(self, nil, where)
+                        else
+                          model.where(where).first
+                        end
```

Comment:
```suggestion
              object = model.where(where).first
```

Why it is not possible? 🤔

Reply:
I don't get this one here. Do you want to remove the resolver hook?